src/ui_widgets/LineEditButton.gd

```diff
@@ -5,8 +5,6 @@ class_name LineEditButton extends Control
 # A fake-out is drawn to avoid adding unnecessary nodes.
 # The real controls are only created when necessary, such as when hovered or focused.
 
-const BUTTON_WIDTH = 14.0
-
```

Comment:
This doesn't seem like it's used in the PR, so I think it should be reverted. It doesn't need to change until needed.